spikeinterface/extractors/neoextractors/spikeglx.py

```diff
@@ -28,22 +28,34 @@ class SpikeGLXRecordingExtractor(NeoBaseRecordingExtractor):
     stream_id: str or None
         stream for instance : 'imec0.ap' 'nidq' or 'imec0.lf'
     """
-    mode = 'folder'
-    NeoRawIOClass = 'SpikeGLXRawIO'
+    mode = "folder"
+    NeoRawIOClass = "SpikeGLXRawIO"
 
     def __init__(self, folder_path, stream_id=None):
-        neo_kwargs = {'dirname': str(folder_path)}
+        neo_kwargs = {"dirname": str(folder_path)}
         if HAS_NEO_10_2:
-            neo_kwargs['load_sync_channel'] = False
+            neo_kwargs["load_sync_channel"] = False
         NeoBaseRecordingExtractor.__init__(self, stream_id=stream_id, **neo_kwargs)
 
-        #~ # open the corresponding stream probe
-        if HAS_NEO_10_2 and '.ap' in self.stream_id:
-            signals_info_dict = {e['stream_name'] : e for e  in self.neo_reader.signals_info_list}
-            meta_filename = signals_info_dict[self.stream_id]['meta_file']
-            probe = pi.read_spikeglx(meta_filename)
+        # ~ # open the corresponding stream probe
+        if HAS_NEO_10_2 and "nidq" not in self.stream_id:
```

Comment:
There was a typo here and the `nidq` was not being kept out from the loop. It should be fine now.